admin/class-primary-term-admin.php

```diff
@@ -28,7 +28,7 @@ public function __construct() {
 	 * @return integer The post ID.
 	 */
 	protected function get_current_id() {
-		return filter_input( INPUT_GET, 'post', FILTER_SANITIZE_NUMBER_INT );
+		return $GLOBALS['post_ID'];
```

Comment:
Can you use `get_the_ID()` for this. By using that core WordPress function we don't have to access `$GLOBALS`